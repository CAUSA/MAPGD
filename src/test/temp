id1		id2	ref	major	minor	cov	M	m	error	null_e	f	MM	Mm	mm	h	poly_ll	hwe_ll	gof	ef_chrm	N	N_cut	model_ll
scaffold_name 	1	C	G	C	150	0.7000	0.3000	0.9000	0.9000	0.6349	0.6000	0.2000	0.2000	0.4667	61.2761	1.6714	-59.8186	10.0000	5	0	-209.2829
@sample_1:-9.1964
@sample_2:-9.3055
@sample_3:-24.7477
@sample_4:-40.1112
@sameple_5:-50.3975
