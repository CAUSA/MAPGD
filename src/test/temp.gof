@NAME:SAMPLE	VERSION:0.4.1
@SMPNAME	GOF
Sample_0	-nan
Sample_1	-nan
Sample_2	-nan
Sample_3	-nan
Sample_4	-nan
Sample_5	-nan
Sample_6	-nan
Sample_7	-nan
@END_TABLE
