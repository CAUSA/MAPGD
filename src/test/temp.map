@NAME:GENOME	VERSION:0.4.1
@ID0    	ID1	REF	MAJOR	MINOR	COVERAG	MJ_FREQ	MN_FREQ	ERROR	NULL_ER	F_STAT	MM_FREQ	Mm_FREQ	mm_FREQ	HETERO	POLY_LR	HWE_LR	GOF	EF_CHRM	IND_INC	IND_CUT	BEST_LL
scaffold_1	1	N	N	N	0	.	.	.	.	.	.	.	.	.	0.0000	0.0000	0.0000	0.0000	0	0	0.0000
scaffold_1	2	N	A	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_1	3	N	A	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_1	4	N	A	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_1	5	N	T	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_1	6	N	C	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_1	7	N	G	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_1	8	N	C	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_1	9	N	C	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_1	10	N	C	N	2	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	2.00	2	0	-0.0020
scaffold_2	1	N	T	N	2	1.0000	0.0000	0.0010	0.0010	1.0000	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.04	1.50	1	0	-0.0020
scaffold_2	2	N	A	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_2	3	N	A	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_2	4	N	A	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_2	5	N	T	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_2	6	N	C	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_2	7	N	G	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_2	8	N	C	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_2	9	N	C	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_2	10	N	C	N	3	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.05	2.50	2	0	-0.0030
scaffold_3	1	N	T	A	3	0.5000	0.5000	0.0010	0.3333	0.0000	0.0000	1.0000	0.0000	1.0000	1.85	1.38	0.58	1.75	1	0	-0.9828
scaffold_3	2	N	A	N	4	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.06	2.75	2	0	-0.0040
scaffold_3	3	N	A	N	4	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.06	2.75	2	0	-0.0040
scaffold_3	4	N	A	N	4	1.0000	0.0000	0.0010	0.0010	0.3333	1.0000	0.0000	0.0000	0.0000	0.00	0.00	0.06	2.75	2	0	-0.0040
scaffold_3	5	N	A	T	4	0.5000	0.5000	0.0010	0.2500	-0.6667	0.0000	1.0000	0.0000	0.6667	9.93	1.38	0.58	2.75	2	0	-1.6766
scaffold_3	6	N	C	A	4	0.5000	0.5000	0.0010	0.2500	-0.6667	0.0000	1.0000	0.0000	0.6667	1.15	1.38	0.58	2.75	2	0	-1.6766
scaffold_3	7	N	G	A	4	0.5000	0.5000	0.0010	0.2500	-0.6667	0.0000	1.0000	0.0000	0.6667	1.15	1.38	0.58	2.75	2	0	-1.6766
scaffold_3	8	N	A	C	4	0.5000	0.5000	0.0010	0.2500	-0.6667	0.0000	1.0000	0.0000	0.6667	9.93	1.38	0.58	2.75	2	0	-1.6766
scaffold_3	9	N	C	A	4	0.5000	0.5000	0.0010	0.2500	-0.6667	0.0000	1.0000	0.0000	0.6667	1.15	1.38	0.58	2.75	2	0	-1.6766
scaffold_3	10	N	C	A	61	0.6250	0.3750	0.0010	0.4918	0.6095	0.5000	0.2500	0.2500	0.5357	114.76	0.39	0.28	7.00	4	0	-5.7260
@END_TABLE
