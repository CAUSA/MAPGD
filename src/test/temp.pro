@NAME:QUARTETS	VERSION:0.4.1
@ID0       	ID1	REF	Sample_1	Sample_2	Sample_3	Sample_4	Sample_5	Sample_6	Sample_7	Sample_8
scaffold_1	1	N	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	2	A	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	3	A	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	4	A	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	5	T	000/000/000/000	000/000/000/001	000/000/000/000	000/000/000/000	000/000/000/001	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	6	C	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	7	G	000/000/000/000	000/000/001/000	000/000/000/000	000/000/000/000	000/000/001/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	8	C	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	9	C	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_1	10	C	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	1	T	000/000/000/000	000/000/000/002	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	2	A	000/000/000/000	002/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	3	A	000/000/000/000	002/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	4	A	000/000/000/000	002/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	5	T	000/000/000/000	000/000/000/002	000/000/000/000	000/000/000/000	000/000/000/001	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	6	C	000/000/000/000	000/002/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	7	G	000/000/000/000	000/000/002/000	000/000/000/000	000/000/000/000	000/000/001/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	8	C	000/000/000/000	000/002/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	9	C	000/000/000/000	000/002/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_2	10	C	000/000/000/000	000/002/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	1	T	000/000/000/000	001/000/000/002	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	2	A	000/000/000/000	003/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	3	A	000/000/000/000	003/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	4	A	000/000/000/000	003/000/000/000	000/000/000/000	000/000/000/000	001/000/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	5	T	000/000/000/000	001/000/000/002	000/000/000/000	000/000/000/000	000/000/000/001	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	6	C	000/000/000/000	001/002/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	7	G	000/000/000/000	001/000/002/000	000/000/000/000	000/000/000/000	000/000/001/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	8	C	000/000/000/000	001/002/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	9	C	000/000/000/000	001/002/000/000	000/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
scaffold_3	10	C	000/020/000/000	010/010/000/000	020/000/000/000	000/000/000/000	000/001/000/000	000/000/000/000	000/000/000/000	000/000/000/000
@END_TABLE
