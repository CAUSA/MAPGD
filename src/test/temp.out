id1	id2	ref	major	minor	cov	M	m	error	null_e	f	MM	Mm	mm	h	polyll	HWEll	gof	eff_chrom	N	N_excluded	model_ll
scaffold_0	2	A	A	*	6	1.0000	0.0000	0.0000	0.0000	0.0000	1.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	2.9375	2	0	0.0000
